src/Bridge/Symfony/Maker/MakeDataPersister.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Maker;
+
+use ApiPlatform\Core\Metadata\Resource\Factory\ResourceNameCollectionFactoryInterface;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Maker\AbstractMaker;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Question\Question;
+
+final class MakeDataPersister extends AbstractMaker
+{
+    private $resourceNameCollection;
+
+    public function __construct(ResourceNameCollectionFactoryInterface $resourceNameCollection)
+    {
+        $this->resourceNameCollection = $resourceNameCollection;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getCommandName(): string
+    {
+        return 'make:data-persister';
+    }
+
```

Comment:
You should add the `getCommandDescription` static method instead of a call to `setDescription` (seems new to make it lazy).